Mudit_Gupta/progress.md

```diff
@@ -3,78 +3,80 @@
 **Project: Usability Hero**
 
 Overall Direction:
-- 
-- 
-- 
+- Provide Context specific help
+- Improve Getting Started guide
+- Improve documentation structure
+- Sidebar for sections of instructor help
+- Resolve UI/UX issues
 
 ---
 
 Week 1 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Work on [Student: feedback submission: distribution points question: explain if equal points are not allowed](https://github.com/TEAMMATES/teammates/issues/8817)
+- [ ] Discuss with Cara further work to be done on Getting started guide and help docs
 
 Week 2 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Work on [New instructors: set the time zone of the sample course to match user](https://github.com/TEAMMATES/teammates/issues/8787)
+- [ ] Front end development for sidebar of instructor help
 
 Week 3 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Instructor: Edit session: Rubric question: add front-end validation for empty weights (issue #8631)
+- [ ] Write tests for sidebar
 
 Week 4 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Buffer period to resolve pending work from week 1-3
+- [ ] Fix issues after code review
 
 Week 5 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Develop use cases/user stories + UI workflows for enhancements to Getting started guide
+- [ ] Mockups for enhancements to Getting started guide
 
 Week 6 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Front end development for enhancements to Getting started guide
+- [ ] Writing Selenium tests
 
 Week 7 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Back end development (if any) for getting started guide
+- [ ] Write back end tests 
 
 Week 8 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Buffer period to resolve pending work from week 5-7
+- [ ] Fix issues after code review
 
 Week 9 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Improve documentation structure (phase 1)
```

Comment:
uhmmm i feel this is a bit too generic... is it possible for u to specify? I know u gotta discuss with @craaaa 
and just out of curiosity why do u put this task at last?

Reply:
Well like i mentioned above, I am trying to find what further to do with the docs. One approach I had is similar to my proposal which is to split the topics into separate pages and instead of collapsing windows use a right hand side bar as shown in my proposal.

Reply:
I put this task at last, because I thought the work I and others will do will probably add more to the docs so it might make sense to work on it when most additions have been made ?